### PR TITLE
2 add footer

### PR DESCRIPTION
**Pull request by khiryshank**
## Ticket Description
<-- Copy and paste or write the tickets description here-->
Screenshot 2023-04-17 at 10 01 31 PM

Add footer according to spec 
## Description of Changes
<-- Brief description of the changes you made, files you touched, etc -->

## Before and After for UI Updates
<-- You should have 2 images minimum, for desktop, 4 for mobile and desktop combined -->

Before:
Screenshot 2023-04-12 at 12 30 36 PM

After
Screenshot 2023-04-17 at 6 37 04 PM
th="685" alt="Screenshot 2023-04-17 at 9 58 05 PM" src="https://user-images.githubusercont
Screenshot 2023-04-17 at 9 57 33 PM
ent.com/43101229/232675445-6980a9ec-e953-4679-a340-b4acd5850995.png">
:

## For PR Reviewer
- [ ] Does this file change the yarn.lock, package.json or package-lock.json file? If so, why?
- [ ] If this pr contains mobile and desktop changes, did you test on IphoneXr and desktop views?
- [ ] Does this file match the related tickets linked figma file, or does it pass the visual smell test?
- [ ] If this file contains javascript, does the javascript pass the smell test?
- [ ] If you don't feel super confident in your review, did you assign someone more senior to double check?

### CODE REVIEW

File: pages/spect2.tsx
Comment (najwanan) [inline pages/spect2.tsx:1]: Looks like this might be a duplicate of index.tsx? Can we remove this?

File: .hintrc
Comment (najwanan) [inline .hintrc:1]: I think this is a file from some sort of utility you're using to help with code hints. Great that you found this, hope it's helping. Let's pull it out of your PR. You can do this by adding this file name ".hintrc" to .gitignore. That way it won't get pushed up to the remote branch.

**Review by najwanan — CHANGES_REQUESTED** [review]
@khiryshank it looks like there are currently some merge conflicts. Before tackling those, I'd suggest the following. 
Pull down the latest changes from main, with the new home page and file structure. 
Rearrange your components (move your footer file into components and then import your footer file into home.tsx) 
Get rid of any extraneous files, like the .hintrc or spect2.tsx

Then once all those changes are pushed up, see if there's still merge conflicts

**Comment by khiryshank** [comment]
Kk, will do so and complete it around 6:30 PST

**Comment by khiryshank** [comment]
I'm no longer seeing the merge conflicts, hoping I resolved everything

File: components/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+import GithubLogo from "../public/github-logo.svg"
+import Image from 'next/image'
+import Link from "next/link";
+export default function Footer() {
+    return (
+
+        <footer className="bg-[#0C9FAA] h-80">
+            
+            
+            {/* <div className='flex-col bg-yellow-600 max-w-3xl mx-auto'> */}
```
Comment (najwanan) [inline components/Footer.tsx:14]: Let's pull these out if not needed.

File: components/Footer.tsx [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+                      />
+                    </a> 
+                    
+                     
+                    
+                    </div>
+                    
+                  </div>
+                  </section> 
+              {/* </div>   */}
```
Comment (najwanan) [inline components/Footer.tsx:111]: Same here, delete if not needed or in use

**Review by najwanan — COMMENTED** [review]
So close! There's a slight deviation right now from the mobile design and that is that it's missing the social links. There are a few approaches to handle this. You could create a seperate mobile nav social component and hide it as necessary. 

I'm actually okay with merging in as (once we delete out the extra commented out lines), knowing that we'll have a follow-up ticket to finalize mobile. 
 
Screenshot 2023-04-19 at 2 49 20 PM

**Review by najwanan — APPROVED** [review]
Looks good to me!

**Pull request merged**